cls/SourceControl/Git/WebUIDriver.cls

```diff
@@ -200,6 +200,42 @@ ClassMethod Uncommitted() As %SystemBase
             }
         }
     }
+
+    do fileToOtherDevelopers.%Set("current user's changes", editedByCurrentUser)
+    do fileToOtherDevelopers.%Set("other users' changes", ##class(SourceControl.Git.Change).GetOtherDeveloperChanges())
+    quit fileToOtherDevelopers
+}
+
+ClassMethod UncommittedWithAction() As %SystemBase
+{
+    do ##class(SourceControl.Git.Change).RefreshUncommitted()
+    do ##class(SourceControl.Git.Utils).GitStatus(.files, 1)
+    set output = ""
+    set key = ""
+
+    set editedByCurrentUser = []
+    set fileToOtherDevelopers = {}
+    for {
+        set key = $order(files(key), 1, fileData)
+        quit:key=""
+
+        set filename = ##class(SourceControl.Git.Utils).FullExternalName(key)
+        if (($ISVALIDNUM(key)) && (files(key) '= "")) {
+            set edit = {}
+            do edit.%Set("file", $listget(fileData, 2))
+            do edit.%Set("action", $listget(fileData, 1))
+            do editedByCurrentUser.%Push(edit)
+        } else {
+            set sc=##class(SourceControl.Git.Change).GetUncommitted(filename, .tAction, .tInternalName, .UncommittedUser, .tSource, .UncommittedLastUpdated)
+            if ($$$ISOK(sc)) && ($data(tAction)&&(UncommittedUser=$username)) {
+                set edit = {}
+                do edit.%Set("file", $listget(fileData, 2))
+            do edit.%Set("action", $listget(fileData, 1))
+            do editedByCurrentUser.%Push(edit)
+            }
+        }
+    }
+
     do fileToOtherDevelopers.%Set("current user's changes", editedByCurrentUser)
```

Comment:
Since we're adding a new API, can we pick a more reasonable key here? e.g., call it "user", "others" rather than having long magic strings? Or better yet, since we have attributes already, do edit.%Set("user",UncommittedUser) and do edit.%Set("mine",UncommittedUser=$username)?